app/src/main/java/pm/gnosis/heimdall/ui/tokens/TokensViewModel.kt

```diff
@@ -1,41 +1,86 @@
 package pm.gnosis.heimdall.ui.tokens
```

Comment:
we could use subpacked for the `list` (or overview, or whatever) and `add` or something.

I did that with the multisig, not sure which one is better